.gitlab-ci.yml

```diff
@@ -52,6 +52,7 @@ nogpu-tests:
   needs: ["lint"]
   tags:
     - timemachine
+    - cpu
```

Comment:
Truly run on a machine without a GPU, and without any of cuda drivers/nvidia-smi/etc